hca/util/__init__.py

```diff
@@ -224,20 +224,34 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 
 
 class _PaginatingClientMethodFactory(_ClientMethodFactory):
-    def iterate(self, **kwargs):
-        page = None
-        while page is None or page.links.get("next", {}).get("url"):
-            page = self._request(kwargs, url=page.links["next"]["url"] if page else None)
-            if page.json().get('results'):
-                for result in page.json()['results']:
+    page = None
+    kwargs = None
+
+    def __iter__(self):
```

Comment:
It doesn't make sense to make all responses iterable, because it's ill defined for some methods and violates the principle of least surprise/is too implicit.